portmaster/install/mobile.md

```diff
@@ -3,25 +3,27 @@ title: Status of Mobile
 layout: base
 ---
 
-The Portmaster is currently not available for any mobile platforms.
-While this will comes with a multitude of challanges, our dream is to support as many mobile platforms as possible.  
-
 This page gives an overview over the current status of supporting mobile platforms.
-If you are interested in what we are currently working on and what is planned for the near future, check out our [next page](https://safing.io/next/).
+
+**⚠️ The Portmaster is currently not available for any mobile platforms.**
+
+While we know supporting mobile will come with a multitude of challenges, it still remains our dream to support many mobile platforms in the future. Currently, these tasks [remain in our backlog](https://safing.io/backlog/).
+
+[You can always see what we are currently working on and what is planned for the near future on the Next Page](https://safing.io/next/).
```

Comment:
I dislike such long links as they decrease readability and do not communicate what exactly one will find when clicking it. I'd recommend only putting the link on `Next Page`.